spec/support/test_app.rb

```diff
@@ -96,13 +100,12 @@ def cap(task, subdirectory=nil)
   end
 
   def run(command, subdirectory=nil)
-    output = nil
     command = "bundle exec #{command}" unless command =~ /^bundle\b/
     dir = subdirectory ? test_app_path.join(subdirectory) : test_app_path
-    Dir.chdir(dir) do
-      output = with_clean_bundler_env { `#{command}` }
+    output, status = Dir.chdir(dir) do
+      with_clean_bundler_env { Open3.capture2e(command) }
```

Comment:
🗒️ Changed from backticks to `capture2e` so that stderr from the `cap` command doesn't clutter the console when running the Cucumber tests.